hera_mc/sys_handling.py

```diff
@@ -220,7 +221,7 @@ def get_fully_connected_location_at_date(self, stn, at_date):
                 station_info.stop_date = fctime['end']
         return station_info
 
-    def get_cminfo_correlator(self):
+    def get_cminfo_correlator(self, hookup_type=None):
```

Comment:
What is the default if it's left as `None`? This should be clarified in docstrings here and for the other functions that have this keyword. The allowed options should also be clarified.

Reply:
I like what you added to this function. Can you add it to other API level functions (i.e. not starting with underscores) that have this keyword?